.claude/docs/pr-testing-agent.md

```
@@ -0,0 +1,891 @@
+bundle exec rake autofix               # Formatting applied
+
+# MUST test in browser:
+cd spec/dummy
+bin/dev                                # Start servers
+# Visit http://localhost:3000/hello_world
+# Open browser console - check for errors
+```
+
+### 3. Build Configuration Changes
+
+**CRITICAL: These changes REQUIRE extensive manual testing**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)